### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by rspt**

### CODE REVIEW

**Comment by maxirozay** [comment]
The menu is too big now, so the french link is on a second line.

**Comment by maxirozay** [comment]
The titles of acceuil, services and processus are in enlgish on the french website.

**Comment by rspt** [comment]
And tadaaaa ;)

**Comment by maxirozay** [comment]
Now the mobile menu on medium screen look like a big blank page without margin we should do something about it in another PR.

**Comment by rspt** [comment]
Yeah, I told you about the big blank screen ;)

But yes I think it's in another PR.

File: _data/lang/fr.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      name: Julien Grünhagel
+      description: >
+        Le fondateur de laruche, celui qui se concentre sur la gestion de
+        projet et le consulting afin de garantir que nous délivrons le meilleur
+        miel de la région.
+    maxime:
+      name: Maxime Rossier
+      description: ???
+    you:
+      name: You?
```
Comment (maxirozay) [inline _data/lang/fr.yml]: Vous?

File: _data/lang/fr.yml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      description: Viens rejoindre les rebelles.
```
Comment (maxirozay) [inline _data/lang/fr.yml]: Viens rejoindre la rebellion.?

File: index.html [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
+languages:
+- en
+- fr
+subset: home
+permalink: /:language/index.html
 ---
 
-{% include line.html line="laruche is a full-service web agency based in Geneva." bg-color="bg-light-red" %}
+{% capture line %}{% t 'line' %}{% endcapture %}
+{% include line.html line=line bg-color="bg-hot-pink" %}
```
Comment (maxirozay) [inline index.html]: What did you do to my beautiful ight-red?!?!

File: _data/lang/fr.yml (hunk 2 above)
Comment (rspt) [inline _data/lang/fr.yml]: Mieux ^^

File: index.html (hunk 3 above)
Comment (rspt) [inline index.html]: It disappear during the merge ^^

**Comment by rspt** [comment]
Done! :)

**Pull request merged**